CHANGELOG.md

```diff
@@ -6,9 +6,27 @@ This change log file is based on best practices from [Keep a Changelog](http://k
 This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.
 This project adheres to the Node [default version scheme](https://docs.npmjs.com/misc/semver).
 
-## [next-version]
```

Comment:
We still want to have the `next-version` header at the top. Clicking on this header will open a link to the list future changes that are in development in comparison to `6.7.0`.

Reply:
Ah, ok. Looks like I misunderstood what you meant in our previous discussion about this. Will bring it back in.